aws/aws_secrets_manager/aws_scm_integration_test.go

```diff
@@ -107,18 +108,27 @@ func (a *awsSecretTest) TestListSecrets(t *testing.T) error {
 
 func (a *awsSecretTest) TestDeleteSecret(t *testing.T) error {
 	// Delete of a key that exists should succeed
-	err := a.s.DeleteSecret(a.secretIdWithData, nil)
+	keyContext := make(map[string]string)
+	keyContext[SecretRetentionPeriodInDaysKey] = "7"
```

Comment:
If you ad a retention period then the subsequent GetSecret check is going to fail ? The integration tests are failing for SecretsManager. Can you check?